packages/edit-post/src/components/sidebar/post-slug/style.scss

```diff
@@ -0,0 +1,12 @@
+.editor-post-slug__input {
+	margin: -5px 0;
+	padding: 2px;
+
+	// Set the width of the author select box in IE11 to prevent it overflowing
+	// outside of the container because of IE11 flexbox bugs.
+	// We reset it to `width: auto;` for non-IE11 browsers.
+	width: 100%;
+	@supports (position: sticky) {
+		width: auto;
+	}
```

Comment:
I think this is a copy/paste error? Seems unnecessary for this input.

Reply:
Oh yes, will remove.